components/Register/index.js

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable jsx-a11y/accessible-emoji */ // FIxed in emoji component
+import { useRouter } from "next/router";
+import {
+  Typography,
+  Form,
+  Input,
+  Button,
+  Select,
+  DatePicker,
+  Upload,
+  Space,
+  Row,
+  Col,
+} from "antd";
+import { UploadOutlined } from "@ant-design/icons";
+import { useAuth } from "~/components/Auth";
+import * as api from "~/lib/utils/api.js";
+import Emoji from "../Emoji";
+
+import styles from "./style.module.css";
+
+const { Option } = Select;
+
+const CountrySelect = () => (
+  <Select defaultValue="+351">
+    <Option value="+351">
+      <Emoji label="Bandeira Portuguesa">🇵🇹</Emoji>
+    </Option>
+  </Select>
+);
+
+function Register() {
```

Comment:
This component should depend on which type of user is trying to register, because it is different from a mentor vs guardian

Reply:
Yeah. I'll open an issue and fix that in the future as I work on the remainder of the sign up flow